lib/shopify_cli/context.rb

```diff
@@ -621,7 +621,10 @@ def which(cmd)
     #
     def new_version
       if (time_of_last_check + VERSION_CHECK_INTERVAL) < (Time.now.to_i)
-        fork do
+        # Fork is not supported in Windows
```

Comment:
Mmm no, the thread will be killed when the command finishes, and this operation could take more time... Maybe it's better to release this simple fix because the CLI is broken for Windows, and then we can try to find a better way.

Reply:
Yeah, but then the command won't finish until it's done, so it's almost the same as doing it inline, right?

Reply:
Good point, let's better use Thread for Windows. I've already tested in all platforms and it works as expected.